### ISSUE TITLE
[course-calendar] Take inspiration from this jeckyll course space

### ISSUE THREAD

**Issue by btopro** [haxtheweb/issues#942]
Stumbled on this through an educator group; would be nice to add to hax as a brick that visualizes the course "outline" as data. It's probably just some weird markdown but looks very clean.

https://dsc10.com/
https://dsc40a.com/

![image](https://user-images.githubusercontent.com/329735/151872451-cdea0ee3-2250-4f52-a515-a514995d44c5.png)

**Comment by btopro** [comment]
@cgldevel from the screenshot, it almost looks like a calendar based version of your group's home page / high level outline thing.

**Comment by cgldevel** [comment]
Nice, very easy to read......and I'll bet it could be wired similarly to our course-intro element (variables in site.json). I also dig the color-coded activities... We're messing w/ intentional colors associated w/ calls-to-action in the instruction-card element... ie: all orange cards are 'Learning Objectives', all teal cards are 'Learning Strategies', etc.

**Comment by btopro** [comment]
I agree. What's interesting to me is that both courses use a different "tag" and that "tag" seems to have different colors associated. I almost wonder if there's sorta a "my 6 meaningful colors" or something to it where you can like make a list of colors you care about and then apply them as the color of record throughout parts of the system (like the tag here and the Learning Objectives card to match). @nikkimk thoughts on the table and if we'd be describing an editable-table sorta thing or if it's an element that reads off data and is generated?
